adapter/src/main/kotlin/de/fraunhofer/iem/kpiCalculator/adapter/tools/trivy/TrivyAdapter.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2024 Fraunhofer IEM. All rights reserved.
+ *
+ * Licensed under the MIT license. See LICENSE file in the project root for details.
+ *
+ * SPDX-License-Identifier: MIT
+ * License-Filename: LICENSE
+ */
+
+package de.fraunhofer.iem.kpiCalculator.adapter.tools.trivy
+
+import de.fraunhofer.iem.kpiCalculator.adapter.AdapterResult
+import de.fraunhofer.iem.kpiCalculator.adapter.KpiAdapter
+import de.fraunhofer.iem.kpiCalculator.adapter.kpis.cve.CveAdapter
+import de.fraunhofer.iem.kpiCalculator.model.adapter.trivy.*
+import de.fraunhofer.iem.kpiCalculator.model.adapter.vulnerability.VulnerabilityDto
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.*
+import java.io.InputStream
+import kotlin.math.max
+
+object TrivyAdapter : KpiAdapter<TrivyDto> {
```

Comment:
Generally I'm missing logging in the adapter. we have the logging library included now so we could communicate to the caller what is happening.
What I'd like to see is
- an info log which Trivy version was identified and is processed.
- maybe a debug log telling us about the processed CVSS objects so we can easily see what the adapter found in the result?